mbtiles/mbtiles.go

```diff
@@ -278,8 +304,43 @@ func (tileset *Mbtiles) ReadMetadata() (map[string]interface{}, error) {
 	return metadata, nil
 }
 
-// Close the MBtiles database connection
-func (tileset *Mbtiles) Close() error {
+// TileFormatreturns the TileFormat of the DB.
+func (d DB) TileFormat() TileFormat {
+	return d.tileformat
+}
+
+// TileFormatString returns the string representation of the TileFormat of the DB.
+func (d DB) TileFormatString() string {
+	return d.tileformat.String()
+}
+
+// ContentType returns the content-type string of the TileFormat of the DB.
+func (d DB) ContentType() string {
+	return d.tileformat.ContentType()
+}
+
+// HasUTFGrid returns whether the DB has a UTF grid.
+func (d DB) HasUTFGrid() bool {
+	return d.hasUTFGrid
+}
+
+// HasUTFGridData returns whether the DB has UTF grid data.
+func (d DB) HasUTFGridData() bool {
+	return d.hasUTFGridData
+}
+
+// UTFGridCompression returns whether the TileFormat of the UTF Grid compression of the DB.
```

Comment:
Comment is not quite correct.  Should be
`UTFGridCompression returns the compression type of the UTFGrid in the DB: ZLIB or GZIP`

Reply:
Thanks for that, I really was just guessing what the method was doing. Will correct right away.